src/features/artReg/components/ArtRegForm.module.css

```diff
@@ -1,3 +1,48 @@
+@font-face {
```

Comment:
I think for setup should be in the [index.css](https://github.com/pastelnetwork/pastel-electron-wallet/blob/master/src/index.css)